source/blueprints/overview.rst

```diff
@@ -68,7 +68,14 @@ Depending on which blueprint you choose to run, this Quick Start should take bet
 1. Download
 ^^^^^^^^^^^
 
-Open a Python shell and type the following commands to download and set up the blueprint application of your choice.
```

Comment:
Type the following commands to download and set up the blueprint application of your choice. Use the appropriate blueprint name as the value of the ``bp_name`` variable (either ``banking_assistant``, ``hr_assistant``, ``food_ordering``, ``home_assistant``, ``video_assistant``, or ``kwik_e_mart``). In these examples, we specify the Food Ordering blueprint.